### PR TITLE
Implement screen functionalies

### LINKED ISSUE #18: [TASK] Make the "Create screen" screen
Depends on #19 
Design:
Screenshot from 2020-03-21 10-43-10

### LINKED ISSUE #19: [TASK] Make the "Select Screen" screen
Design:
Screenshot from 2020-03-21 10-44-13

### PR DESCRIPTION
**Pull request by ACmiester**
This PR closes #18 #19 
However, I have not linked a Screen up to a task, as I do not know how we should link the selector screen up.
Therefore I propose that we get this PR in, and then make the connection and finalize it. If you don't like this, let me know, and I will work on it further.

### CODE REVIEW

File: application/lib/view/screens/ScreenSelectionScreen.dart [hunk 1]
```diff
@@ -1,9 +1,154 @@
+import 'package:application/blocs/ScreenBloc.dart';
+import 'package:application/di.dart';
+import 'package:application/model/Application.dart';
+import 'package:application/model/Screen.dart';
 import 'package:application/view/screens/BaseScreen.dart';
+import 'package:application/view/screens/NewScreenScreen.dart';
 import 'package:flutter/material.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+
+import '../../routes.dart';
```
Comment (Tricky12321) [inline application/lib/view/screens/ScreenSelectionScreen.dart]: Det skal være import "'package:........routes.dart'" og ikke "import '../../routes.dart'"

File: application/lib/view/screens/ScreenSelectionScreen.dart [hunk 2]
```diff
@@ -1,9 +1,154 @@
...
+        .getScreens(application.id)
+        .then((value) => value.forEach((element) {
+              screens.add(element);
+            }));
+  }
+  final ScreenBloc screenBloc = di.getDependency<ScreenBloc>();
+
+  final Application application;
+
+  int screensEachRowPortrait() => isTablet() ? 3 : 3;
```
Comment (Tricky12321) [inline application/lib/view/screens/ScreenSelectionScreen.dart]: Skal der være lige mange om det er tablet eller ej?

File: application/lib/model/Screen.dart [hunk 3]
```diff
@@ -1,14 +1,18 @@
+import 'dart:convert';
+
```
Comment (Tricky12321) [inline application/lib/model/Screen.dart]:
```suggestion
```

**Review by Tricky12321 — CHANGES_REQUESTED** [review]
Lige enkelte ting

File: application/lib/view/screens/ScreenSelectionScreen.dart (hunk 1 above)
Comment (ACmiester) [reply application/lib/view/screens/ScreenSelectionScreen.dart]: Har ændret det.
Hvad er egentlig forskellen?

File: application/lib/view/screens/ScreenSelectionScreen.dart (hunk 2 above)
Comment (ACmiester) [reply application/lib/view/screens/ScreenSelectionScreen.dart]: Fixed

**Review by Tricky12321 — APPROVED** [review]

**Pull request merged**